app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
+script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"
+script src="http://www.matchingnotes.com/javascripts/leaflet-google.js"
+
+h1 Мапа звернень
+#issues-map
+
+javascript: 
+  $(document).ready(function(){
```

Comment:
to many logic in html
create JS file and just initialize your new function from here